quattor/types/system.pan

```diff
@@ -27,7 +27,7 @@ type structure_oldname = {
 include 'components/network/core-schema';
 
 # TODO: monitoring to be defined
-include 'monitoring/lemon/schema';
+include { if_exists('monitoring/lemon/schema') };
```

Comment:
@samary the simplest solution here if for you to just delete this line and the one above and add
```pan
bind /system/monitoring = structure_monitoring;
```
to `monitoring/lemon/schema.pan`.